src/test/kotlin/apply/ui/api/EvaluationTargetRestControllerTest.kt

```diff
@@ -225,6 +228,42 @@ internal class EvaluationTargetRestControllerTest : RestControllerTest() {
             )
     }
 
+    @ParameterizedTest
+    @EnumSource(names = ["PASS", "FAIL", "WAITING"])
```

Comment:
변경 가능성이 높은 어노테이션을 맨 위에 배치해 보죠.

```suggestion
    @EnumSource(names = ["PASS", "FAIL", "WAITING"])
    @ParameterizedTest
```